lite2/client.go

```diff
@@ -415,25 +415,30 @@ func (c *Client) TrustedHeader(height int64, now time.Time) (*types.SignedHeader
 	if height < 0 {
 		return nil, errors.New("negative height")
 	}
-
+	latestHeight, err := c.LastTrustedHeight()
```

Comment:
So just to clarify here, this will fetch a header from a provider if there are no headers in the DB?

Reply:
Yes but only if the requested header is between the first and last trusted header heights in the db. i.e. if you call one that is more recent you will need to call an update first